timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -398,9 +397,14 @@ def _add_gaussian_quantiles(self, predictions: pd.DataFrame, repeated_item_ids:
         normal_scale_per_timestep = pd.Series(np.tile(sqrt_h, num_items), index=repeated_item_ids)
 
         residuals_std_per_timestep = self._residuals_std_per_item.reindex(repeated_item_ids)
-        # Use avg_residuals_std in case unseen item received for prediction
-        if residuals_std_per_timestep.isna().any():
-            residuals_std_per_timestep = residuals_std_per_timestep.fillna(value=self._avg_residuals_std)
+        # Use in-sample seasonal error in for items not seen during fit
+        items_not_seen_during_fit = residuals_std_per_timestep.index[residuals_std_per_timestep.isna()].unique()
+        if len(items_not_seen_during_fit):
+            scale_for_new_items = np.sqrt(
```

Comment:
Will this give a single scale across all new items or 1 scale per unique item?

Reply:
This returns a `pd.Series`, with `item_id` as index and the scale of the historic errors as value.

Added a type hint to make this more explicit.